chain/network/src/routing/actor.rs

```diff
@@ -65,6 +65,15 @@ impl Actor {
         }
     }
 
+    pub fn spawn(
+        clock: time::Clock,
+        store: store::Store,
+        graph: Arc<RwLock<routing::GraphWithCache>>,
+    ) -> actix::Addr<Self> {
+        let arbiter = actix::Arbiter::new();
```

Comment:
I am forgetting, how does thread shurtdown works? Would it automatically die once there are no actors, or should we request it to die explicitly? 

It feels a bit strange that we do `actix::Arbiter::current().stop();` for PeerManager and Peer, but not for routing::Actor?

Reply:
Stopping the actor doesn't stop the arbiter. AFAIK stopping arbiter is necessary to avoid leaking a runtime. I'll add Arbiter::current().stop() to routing::Actor.